apps/assisted_tagging/serializers.py

```diff
@@ -127,3 +127,51 @@ def validate(self, data):
                 association='Association is required for this widget.'
             ))
         return data
+
+
+class AutoDraftEntryGqlSerializer(ProjectPropertySerializerMixin, UserResourceCreatedMixin, serializers.ModelSerializer):
```

Comment:
This is a bit confusing, are we trigger auto draft entries for lead or updating draft entry?
We should have different mutations for each one.